app/src/androidTest/java/testUtils/TestConstants.java

```diff
@@ -32,9 +32,8 @@ When we call startForgroundSerivce() method, Service do not get started immediat
 
     public static boolean WAKE_LOCK_ENABLED = true;
     public static boolean WAKE_LOCK_DISABLED = false;
-    public static boolean ALARM_LOCK_ENABLED = true;
-    public static boolean ALARM_LOCK_DISABLED = false;
     public static int ALARM_TRIGGER_INTERVAL = 15000;
+    public static int INACTIVITY_INTERVAL = 5 * 60 * 1000;
```

Comment:
whats this?

Reply:
Dev team has added new param which will help to reduce api calls if driver hasn't moved in given time which is `inactivityInterval`. 
`5 * 60 * 1000` means 5 minutes in milli seconds. 
They wanted to make it server configurable therefore created new param while initialising location sdk. 
We have kept it `5 * 60 * 1000` to match up the defaults.